.github/workflows/winget-submission.yml

```diff
@@ -0,0 +1,23 @@
+name: Submit package to Windows Package Manager Community Repository 
+
+on:
+  release:
+    types: [published]
+
+env:
+  version: "$(git describe --tags --abbrev=0)"
+  packageFileName: coder_${version}_windows_amd64.zip
+  packageId: Coder.Coder  # Most probably the id of coder package
+  
+jobs:
+  
+  winget:
+    name: Publish winget package
+    runs-on: windows-latest
+    steps:
+      - name: Submit package to Windows Package Manager Community Repository
+        run: |
```

Comment:
`set -euo pipefail`

Reply:
I did not understand this comment.

Reply:
Looking back on this comment it doesn't really make sense since this is powershell.... oops. `set -euo pipefail` is a command in bash (not powershell lol) that tells bash to exit on error, fail on unbound variables, and error when any command in a pipe fails.

Disregard this comment since it doesn't apply to powershell

Reply:
yes, that's what got me confused. Anyway, I opened a new PR at #3729. you may look at that sometimes.